etc/kayobe/kolla/config/haproxy/services.d/os_exporter.cfg

```diff
@@ -7,6 +8,9 @@ frontend os_capacity_frontend
     http-request set-header X-Forwarded-Proto https if { ssl_fc }
     bind {{ kolla_internal_vip_address }}:9000
     default_backend os_capacity_backend
+    {% if kolla_enable_tls_internal | bool %}
+    scheme: https
+    {% endif %}
```

Comment:
This is the wrong place for this config. It needs to go in prometheus scrape config, etc/kayobe/kolla/config/prometheus/prometheus.yml.d/70-oscapacity.yml.